x-pack/plugins/security_solution/server/lists_integration/endpoint/validators/blocklist_validator.ts

```diff
@@ -260,7 +264,7 @@ export class BlocklistValidator extends BaseValidator {
 
     await this.validateByPolicyItem(updatedItem);
 
-    return updatedItem as UpdateExceptionListItemOptions;
+    return _updatedItem;
```

Comment:
Thanks for updating this. 

Could you also change this in [trusted app validator](https://github.com/elastic/kibana/blob/27e170a6649367137a241b24f5fef8940a3712bd/x-pack/plugins/security_solution/server/lists_integration/endpoint/validators/trusted_app_validator.ts#L233)  🙏

Reply:
Will merge this and will create another pr with that change if it's ok to you 🙂

Reply:
Done in https://github.com/elastic/kibana/pull/128252/commits/ba3c8ae42ebfa9bc069715433a39ec6baec30d0f

Reply:
@ashokaditya ^

Reply:
Thanks for this change 🙏